linera-core/src/local_node.rs

```diff
@@ -221,8 +227,128 @@ where
         .collect::<Vec<_>>()
     }
 
+    pub async fn find_missing_application_bytecodes(
```

Comment:
I noticed we have three `find_missing_application_bytecodes` and three `find_missing_blobs` in the code. Are these really all different? Is there an opportunity to deduplicate something?

Reply:
Right, I missed cleaning those up too, will do that!